mteb/models/model_meta.py

```diff
@@ -57,6 +57,8 @@
     "ColPali",
 ]
 
+MODEL_TYPES = Literal["dense", "cross_encoder", "late_interaction"]
```

Comment:
minor, but I think this is the general pattern in e.g. licenses:
```suggestion
MODEL_TYPES = Literal["dense", "cross-encoder", "late-interaction"]
```

Reply:
@ayush1298 I think this one is still missing otherwise I think this PR is ready

Reply:
Ohh, odd - I must have checked the historic file